### ISSUE TITLE
Fixed baseline and densitiy in cudaprob

### ISSUE THREAD

**Issue by KSkwarczynski** [mach3-software/MaCh3#107]
With CUDAProb it is being set at constructor level
https://github.com/mach3-software/MaCh3/blob/29484d5fe402550d945299d79fb86220ad2fc280/samplePDF/samplePDFFDBase.cpp#L940-L941

while in prob3++ etc
https://github.com/mach3-software/MaCh3/blob/29484d5fe402550d945299d79fb86220ad2fc280/samplePDF/samplePDFFDBase.cpp#L224
https://github.com/mach3-software/MaCh3/blob/29484d5fe402550d945299d79fb86220ad2fc280/samplePDF/samplePDFFDBase.cpp#L238

tl:dr prob3++ can use baseline and densitiy as fit parameters, while currently cuda prob3 not
